scripts/build.sh

```diff
@@ -25,22 +25,9 @@ case "$1" in
     cd src/delegation_backend
     LD_LIBRARY_PATH="$OUT" $GO test
     ;;
-  docker-run)
-    if [[ "$GOOGLE_APPLICATION_CREDENTIALS" == "" ]]; then
-      echo "Specify path to credentials JSON file in env variable GOOGLE_APPLICATION_CREDENTIALS"
-      exit 1
-    fi
-    tag=delegation-backend-test
-    docker build -t "$tag" -f ../../../dockerfiles/Dockerfile-delegation-backend .
-    docker run -p 8080:8080 -v "$GOOGLE_APPLICATION_CREDENTIALS":/creds.json -e GOOGLE_APPLICATION_CREDENTIALS=/creds.json "$tag"
-    ;;
-  docker-toolchain | docker)
-    if [[ "$VERSION" == "" ]]; then
-      echo "Specify VERSION"
-      exit 1
-    fi
-    cd ../../..
-    scripts/release-docker.sh -s delegation-backend${1:6} -v "$VERSION"
+  docker)
+    tag=delegation-backend
```

Comment:
Here, instead of hardcoding the value of `tag` I would like to be able to pass down the tag with an environment variable just as before you could do it with `$VERSION`.  I suggest we enter the environment variable and check for the existence of it.